pkg/docker/config/config.go

```diff
@@ -66,7 +67,10 @@ func GetAuthentication(sys *types.SystemContext, registry string) (string, strin
 	dockerLegacyPath := filepath.Join(homedir.Get(), dockerLegacyCfg)
 	pathToAuth, err := getPathToAuth(sys)
 	if err != nil {
-		return "", "", err
+		// Error means that the path set for XDG_RUNTIME_DIR does not exist
+		// but we don't want to completely fail in the case that the user is pulling a public image
+		// Logging the error as a warning instead and moving on to pulling the image
+		logrus.Warnf("%v: Trying to pull image in the event that it is a public image.", err)
```

Comment:
For this part, should I immediately `return "", "", nil` after logging the warning? Or should it go ahead and check the other (docker) auth files to see if it can find creds?

Reply:
How about

```
	paths := [3]string{filepath.Join(homedir.Get(), dockerCfg, dockerCfgFileName), dockerLegacyPath}
 	pathToAuth, err := getPathToAuth(sys)
 	if err == nil {
                paths = append(pathToAuth, paths)
       else {
        	logrus.Warnf("%v: Trying to pull image in the event that it is a public image.", err) 
      }
```

Reply:
I think you should check for the docker auth files as well.

Reply:
okay, fixed.